_episodes/08-conflict.md

```diff
@@ -0,0 +1,338 @@
+
+~~~
+$ git add mars.txt
+$ git commit -m "Add a line about the temperature on Mars"
+~~~
+{: .bash}
+
+~~~
+[master 07ebc69] Add a line in my copy
+ 1 file changed, 1 insertion(+)
+~~~
+{: .output}
+
+Great, now that we've added changes about the temperature
+we can merge them into the `master` branch. First, let's checkout the
+`master` branch.
+
+~~~
+$ git checkout master
+$ git branch
+~~~
+{: .bash}
+
+~~~
+  marsTemp
+* master
+~~~
+{: .output}
+
+And then merge the changes from `marsTemp`
+
+~~~
+$ git merge marsTemp
+~~~
+{: .bash}
+
+~~~
+On branch master
+You have unmerged paths.
+  (fix conflicts and run "git commit")
+
+Unmerged paths:
+  (use "git add <file>..." to mark resolution)
+
+  both modified:   mars.txt
+
+no changes added to commit (use "git add" and/or "git commit -a")
+
+~~~
+{: .output}
+
+![The Conflicting Changes](../fig/conflict.svg)
+
+Git detects that the changes made in one copy overlap with those made
+in the other and stops us from trampling on our previous work. It also
+marks that conflict in the affected file:
+
+~~~
+$ cat mars.txt
+~~~
+{: .bash}
+
+~~~
+Cold and dry, but everything is my favorite color
+The two moons may be a problem for Wolfman
+But the Mummy will appreciate the lack of humidity
+<<<<<<< HEAD
+I'll be able to get 40 extra minutes of beauty rest
+=======
+Yeti will appreciate the cold
+>>>>>>> dabb4c8c450e8475aee9b14b4383acc99f42af1d
+~~~
+{: .output}
+
+Our change—the one in `HEAD`—is preceded by `<<<<<<<`.
+Git has then inserted `=======` as a separator between the conflicting changes
+and marked the end of the content downloaded from GitHub with `>>>>>>>`.
+(The string of letters and digits after that marker
+identifies the commit we've just downloaded.)
+
+It is now up to us to edit this file to remove these markers
+and reconcile the changes.
+We can do anything we want: keep the change made in the local repository, keep
+the change made in the remote repository, write something new to replace both,
+or get rid of the change entirely.
+Let's replace both so that the file looks like this:
+
+~~~
+$ cat mars.txt
+~~~
+{: .bash}
+
+~~~
+Cold and dry, but everything is my favorite color
+The two moons may be a problem for Wolfman
+But the Mummy will appreciate the lack of humidity
+We removed the conflict on this line
+~~~
+{: .output}
+
+To finish merging,
+we add `mars.txt` to the changes being made by the merge
+and then commit:
+
+~~~
+$ git add mars.txt
+$ git status
+~~~
+{: .bash}
+
+~~~
+On branch master
+All conflicts fixed but you are still merging.
+  (use "git commit" to conclude merge)
+
+Changes to be committed:
+
+	modified:   mars.txt
+
+~~~
+{: .output}
+
+~~~
+$ git commit -m "Merge changes from marsTemp"
+~~~
+{: .bash}
+
+~~~
+[master 2abf2b1] Merge changes from marsTemp
+~~~
+{: .output}
+
+Git keeps track of what we've merged with what,
+so we don't have to fix things by hand again. If
+we make another change to the `marsTemp` branch
+
+~~~
+$ git checkout marsTemp
```

Comment:
I sympathize 😆 .  You caught me being lazy in my sections too.  
I don't think it should add to the length of teaching the lesson but it will add to the written lesson of course.  It is up to you though because we can't be sure anyone will use these lessons besides us a this moment, so it isn't vital.  I think splitting it out would be most useful for someone new teaching it or if a learner goes through the materials on their own.